doc/reference/reference_lua/config.rst

```diff
@@ -205,6 +227,65 @@ config API
                   alerts: []
                 ...
 
+    .. _config_api_reference_instance_uri:
+
+    ..  method:: instance_uri([uri_type, opts])
+
+        **Since:** :doc:`3.1.0 </release/3.1.0>`
+
+        Get a URI of the current or remote instance.
+
+        :param string uri_type: a URI type. There are the following URI types are supported:
+
+                                * ``peer`` -- a URI used to advertise the instance to other cluster members. See also: :ref:`iproto.advertise.peer <configuration_reference_iproto_advertise_peer>`.
+                                * ``sharding`` -- a URI used to advertise the current instance to a router and rebalancer. See also: :ref:`iproto.advertise.sharding <configuration_reference_iproto_advertise_sharding>`.
+
+        :param table opts: options to pass. The following options are available:
+
+                           -   ``instance`` -- the name of a remote instance whose URI should be obtained
+
+        :return: an instance URI
```

Comment:
I agree that all the details how the URI is constructed are redundant here. I think I should create a section in the internals and link it from here. It's on me.

But I would add information about the fields in the returned table right into the reference: it is part of the method contract. Also, it worth to add a note that the return value is suitable to pass into `net_box.connect()` (otherwise it may look logical to extract the `uri` field from it before calling `net_box.connect()`).